fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -85,7 +86,7 @@ def test_create_text_object_and_download_it_from_container(self):
 
         suffix = datetime.utcnow().strftime('%Y%m%d%H%M%S%m')
         container_name = TEST_CONTAINER_PREFIX + suffix
-        text_object_name = TEST_TEXT_OBJECT_PREFIX + suffix + TEST_TEXT_FILE_EXTENSION
+        text_object_name = self.region_name + TEST_TEXT_OBJECT_PREFIX + suffix + TEST_TEXT_FILE_EXTENSION
```

Comment:
Please use region name **after** the prefix, so that all resources share the same prefix. And please use separators like "_" between the name parts: `prefix _ region _ suffix . extension`